recipe/meta.yaml

```
@@ -37,9 +37,36 @@ test:
 
 about:
   home: https://pip.pypa.io
-  license: MIT
-  license_family: MIT
-  license_file: LICENSE.txt
+  license_family: Other
+  license: BSD-2-Clause AND LGPL-2.1 AND Apache-2.0 AND EPL-2.0 AND MIT AND BSD-3-Clause
```

```suggestion
  license: BSD-2-Clause AND LGPL-2.1-or-later AND Apache-2.0 AND EPL-2.0 AND MIT AND BSD-3-Clause AND MPL-2.0 AND ISC
```
- `src/pip/_vendor/certifi/LICENSE` is [MPL-2.0](https://github.com/pypa/pip/blob/24.0/src/pip/_vendor/certifi/LICENSE#L16-L18)
- `src/pip/_vendor/chardet/LICENSE` is [LGPL-2.1-or-later](https://github.com/pypa/pip/blob/24.0/src/pip/_vendor/chardet/LICENSE)
- `src/pip/_vendor/resolvelib/LICENSE` is [ISC](https://github.com/pypa/pip/blob/24.0/src/pip/_vendor/resolvelib/LICENSE)

See also https://fossies.org/linux/misc/pip-24.0.tar.gz/meta.html

However, maintaining this list is quite burdensome

---

change reverted to avoid confusion